apps/front-client/src/features/profile/hooks/useUpdateUser.ts

```diff
@@ -0,0 +1,37 @@
+import { useMutation } from '@tanstack/react-query';
+import { User } from 'next-auth';
+import { useSession } from 'next-auth/react';
+import { IUpdateUserForm } from '../types';
+
+const updateUser = async (
+  id: string,
+  updateUserForm: IUpdateUserForm,
+  access_token?: string
+): Promise<User> => {
+  const response = await fetch(`${process.env.API_URL}/user/${id}`, {
```

Comment:
Okay alors tu peux éventuellement tester ça : dans ton next config remplace :
```ts
  env: {
    API_URL: process.env.API_URL + '/api' || 'http://localhost:3000/api',
  },
```
par :
```ts
  env: {
    API_URL: process.env.API_URL || 'http://localhost:3000',
  },
```

Et ensuite ici
```suggestion
  const url = new URL(`/api/user/${id}`, process.env.API_URL);
  const response = await fetch(url, {
```